asn1ate/sema.py

```diff
@@ -719,6 +721,18 @@ def __str__(self):
     __repr__ = __str__
 
 
+class Imports(SemaNode):
+    def __init__(self,elements):
+	self.module2symbols = { }
+	for i in range (0,len(elements),2):
+		self.module2symbols [elements [i+1].elements[0]] = set (elements [i])
+	print 'Imported from', self.module2symbols.keys ()
+
+class Exports(Imports):
```

Comment:
Agreed, this is not conceptual but merely to save repeating code (and perhaps some maintenance on it).  I copied the bit of code for Imports to Exports.  I am well aware that there are multilple ways of viewing inheritance -- conceptual enhancements, implementation enhancements, specialisation, refinement... and agree with your preference for the first.